GIT_AND_GITHUB.md

```diff
@@ -47,19 +47,30 @@ Use `--no-ff` when you merge to `master`.
 It's okay to just push to `master` if it's a quick fix and you really need
 it out the door. Otherwise, make it a pull request.
 
+### Pull requests
+
 Pull request descriptions should be concise and well written. The merger should
 be able to copy this description straight into the release notes instead of
 figuring out what changed or was fixed.
 
-#### Who merges the pull request?
+Besides a clear description more information could be helpful, for example:
+
+- If is static UI, a screenshot would do. 
+- If is an interaction a gif would help a lot. A good tool to make gifs is [Licecap](http://www.cockos.com/licecap/).
+- If is a bug, steps to reproduce are useful to understand the context.
+
+#### When to merge a pull request?
+
+Before merging a pull request your code has to be reviewed so we can learn from you, 
```

Comment:
The ending dot after `you` might not be right since they are 3 connected reasons.

- learn from you
- point out your silly mistakes
- add the sufficient amount of gifs